content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,159 @@
+#. Go to :menuselection:`Settings --> API access` in the navigation menu.
+
+Your POS APIs credentials are located in the :guilabel:`POS APIs Credential` section. Click
+:guilabel:`Generate a pair of credentials` to generate them. Save the keys and paste them into the
+Odoo :guilabel:`Client secret` and :guilabel:`Client ID` fields :ref:`when creating a payment method
+<viva_wallet/method-creation>`.
+
+.. image:: viva_wallet/api-cred.png
+   :alt: Client secret and client ID fields
+
+.. warning::
+   These credentials are only displayed once. Ensure you keep a copy to secure them.
+
+.. note::
+   These credentials are used for Android and iOS POS Activation requests, as well as the Cloud
+   Terminal API.
+
+.. seealso::
+   `Find your POS APIs Credentials
+   <https://developer.vivawallet.com/getting-started/find-your-account-credentials/pos-apis-credentials/>`_.
+
+.. _viva_wallet/identifier:
+
+Terminal ID
+~~~~~~~~~~~
+
+To find your terminal ID number:
+
+#. Go to your Viva Wallet account and select the relevant account.
+#. Go to :menuselection:`Sales --> Physical payments --> Card terminals` in the navigation menu.
+
+The terminal ID number is located under the :guilabel:`Terminal ID (TID)` column.
+
+.. image:: viva_wallet/terminal-id.png
+   :alt: Viva terminal ID
+
+.. _viva_wallet/method-creation:
+
+Configure the payment method
+----------------------------
+
+Enable the payment terminal :ref:`in the application settings <configuration/settings>` and
+:doc:`create the related payment method <../../payment_methods>`. Set the journal type as
+:guilabel:`Bank` and select :guilabel:`Viva Wallet` in the :guilabel:`Use a Payment Terminal` field.
+
+Finally, fill in the mandatory fields with your
+
+- :ref:`Merchant ID and API key <viva_wallet/ID-key>`
+- :ref:`Client ID and Client secret <viva_wallet/pos-apis>`
+- :ref:`Terminal ID <viva_wallet/identifier>`
+
+.. image:: viva_wallet/create-method-viva-wallet.png
+   :alt: payment method creation form
+   :scale: 75%
+
+Then, save the form and copy the generated webhook URL of the :guilabel:`Viva Wallet Webhook
+Endpoint` field.
+
+Configure the Webhook
+---------------------
+
+Webhooks allow you to receive real-time notifications whenever a transaction occurs within your Viva
+Wallet account. To set them up:
+
+#. Go to :menuselection:`Settings --> API Access --> Webhooks` in your Viva Wallet account.
+#. Click :guilabel:`Create Webhook`.
+
+   .. image:: viva_wallet/register-webhook.png
+      :alt: webhook creation page
+
+#. Enter the previously generated Webhook URL in the :guilabel:`URL` field. Click :guilabel:`Verify`
+   to confirm its validity.
+#. From the :guilabel:`Event Type` dropdown menu, select :guilabel:`Transaction Payment Created`.
+#. Enable notifications by checking the :guilabel:`Active` checkbox.
+#. Click :guilabel:`Save` to complete the configuration.
+
+   .. image:: viva_wallet/webhook-form.png
+      :alt: webhook creation popup window
```

Comment:
As per my previous comment, I would not describe the steps required in Viva. Instead, I would refer to the documentation available on Viva.com and explain the next steps in Odoo.